hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/OmMetadataManagerImpl.java

```diff
@@ -978,20 +977,22 @@ public boolean isBucketEmpty(String volume, String bucket)
    */
   private boolean isKeyPresentInTableCache(String keyPrefix,
                                            Table table) {
```

Comment:
```suggestion
                                           Table<String, ?> table) {
```

Note: caller should also be changed to pass `getDirectoryTable()` instead of `dirTable`.